app/models/assignment_repo.rb

```diff
@@ -30,13 +38,13 @@ def private?
   # Public
   #
   def github_team_id
-    repo_access.github_team_id
+    repo_access.present? ? repo_access.github_team_id : nil
```

Comment:
I cringe at returning `nil` in cases like this. Is there a case when you are calling this method and expecting `nil`? If not, I'd raise an exception here so it goes into our logs and we can fix those calls.

If there are cases where you're expecting `nil`, you'd be better served by returning an instance of a specific class.